data/fields/craft.json

```diff
@@ -39,6 +41,7 @@
             "oil_mill": "Oil Mill",
             "painter": "Painter",
             "parquet_layer": "Parquet Layer",
+            "pest_control": "Pest Control",
```

Comment:
"Pest Control" what kind of?

unqualified "Pest Control" may refer to shop selling products related to it, office of company doing something pest-control related etc

Reply:
https://www.wikidata.org/wiki/Q1723161